Source/Core/Core/CMakeLists.txt

```diff
@@ -144,6 +144,7 @@ add_library(core
   HW/WiimoteEmu/MotionPlus.cpp
   HW/WiimoteEmu/Speaker.cpp
   HW/WiimoteEmu/Extension/Classic.cpp
+  HW/WiimoteEmu/Extension/DrawsomeTablet.cpp
```

Comment:
Please add the header file to the sources as well

Reply:
Right there in the same list? The other headers aren't here (yet?)?